dask_sql/physical/rel/logical/offset.py

```diff
@@ -0,0 +1,113 @@
+from typing import TYPE_CHECKING
+
+import dask.dataframe as dd
+
+from dask_sql.datacontainer import DataContainer
+from dask_sql.physical.rel.base import BaseRelPlugin
+from dask_sql.physical.rex import RexConverter
+from dask_sql.physical.utils.map import map_on_partition_index
+
+if TYPE_CHECKING:
+    import dask_sql
+    from dask_planner.rust import LogicalPlan
+
+
+class DaskOffsetPlugin(BaseRelPlugin):
+    """
+    Offset is used to modify the effective expression bounds in a larger table
+    (OFFSET).
+    """
+
+    class_name = "Offset"
+
+    def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContainer:
+        (dc,) = self.assert_inputs(rel, 1, context)
+        df = dc.df
+        cc = dc.column_container
+
+        offset_node = rel.offset()
+
+        offset = offset_node.getOffset()
+        if offset:
+            offset = RexConverter.convert(rel, offset, df, context=context)
+
+        # TODO: `Fetch` is not currently supported
+        # end = offset_node.getFetch()
```

Comment:
Is this the logic implemented in limit, or something else?

Reply:
Yes, implemented as part of Limit now. I will remove the comment.